lerna.json

```diff
@@ -1,7 +1,12 @@
 {
   "lerna": "2.4.0",
   "version": "independent",
-  "npmClient": "jlpm",
+  "npmClient": "yarn",
```

Comment:
Is there any reason to use `yarn` and not `npm` here?

Reply:
I'd rather just use npm for the final publish stage if we do that (i.e., set npmClient just in the publish stage). We need to use yarn in other stages because we are using workspaces, etc.

Reply:
but I'd still rather use yarn because we have a specific version of yarn shipped, so it's more uniform than calling out to whatever npm people have.

Reply:
Are we using workspaces for any of the things that lerna manages? I'm just trying to get a handle on how deeply lerna gets into the process.

Reply:
yes, workspaces is how the bootstrap process works.

Reply:
IIRC

Reply:
So then how does it work when `npm` is set as the `npmClient`?

Reply:
Since lerna also has different behavior when the `npmClient` string matches `yarn` in the bootstrap command, this will also change the bootstrap process (in ways that I don't really understand). I just tried the bootstrap with this pr, and it appeared to still work correctly, but I wanted to flag it.